tests/workflows/integration_tests/execution/test_workflow_with_classification_single-label_visualization.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+
+from inference.core.env import WORKFLOWS_MAX_CONCURRENT_STEPS
+from inference.core.managers.base import ModelManager
+from inference.core.workflows.core_steps.common.entities import StepExecutionMode
+from inference.core.workflows.execution_engine.core import ExecutionEngine
+from tests.workflows.integration_tests.execution.workflows_gallery_collector.decorators import (
+    add_to_workflows_gallery,
+)
+
+
+WORKFLOW_DEFINITION = {
+  "version": "1.0",
+  "inputs": [
+    {"type": "WorkflowImage","name": "image"},
+    {
+        "type": "WorkflowParameter",
+        "name": "model_id",
+        "default_value": "fruit-ee3k2/1"
+    }
+  ],
+  "steps": [
+    {
+      "type": "roboflow_core/roboflow_classification_model@v1",
+      "name": "model",
+      "images": "$inputs.image",
+      "model_id": "$inputs.model_id"
+    },
+    {
+      "type": "roboflow_core/classification_label_visualization@v1",
+      "name": "classification_label_visualization",
+      "image": "$inputs.image",
+      "predictions": "$steps.model.predictions",
+      "num_classifications": "$inputs.num_classifications",
+      "text": "Class and Confidence",
+      "color_axis": "INDEX",
+      "color_palette": "ROBOFLOW",
+      "text_scale": 1,
+      "text_color": "BLACK",
+      "text_padding": 28,
+      "task_type": "single-label",
+    }
+  ],
+  "outputs": [
+    {
+      "type": "JsonField",
+      "name": "model_predictions",
+      "coordinates_system": "own",
+      "selector": "$steps.model.predictions"
+    },
+    {
+      "type": "JsonField",
+      "name": "classification_label_visualization",
+      "selector": "$steps.classification_label_visualization.image"
+    }
+  ]
+}
+
+@add_to_workflows_gallery(
+    category="Workflows with single-label classification",
+    use_case_title="Workflow with single-label classification label visualization",
+    use_case_description="""
+This workflow demonstrates how to visualize the predictions of a single-label classification model. 
+It is compatable with single-label and multi-label classification tasks. It is also 
+compatible with supervision visualization fields like text position, color, scale, etc.
+    """,
+    workflow_definition=WORKFLOW_DEFINITION,
+)
```

Comment:
place `workflow_name_in_app` so that it gets published with embedded workflow - name should be unique - look for other ocurrences